### PR TITLE
Resolving #56 and #55

### PR DESCRIPTION
**Pull request by Sonichigo**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -13,3 +13,19 @@ fixes # <!-- Creating an issue before an Pull Request never hurts -->
 ## How Has This Been Tested?
 
 ## Screenshots (if appropriate):
+=======
```
Comment (ghost) [inline .github/pull_request_template.md]: what's this?

File: .github/funding.yml [hunk 2]
```diff
@@ -1 +1 @@
-patreon: krishdevdb
+patreon: krishdevdb
```
Comment (ghost) [inline .github/funding.yml]: We actually have remove that

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -50,4 +104,4 @@ body:
       
           Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
     validations:
-        required: false
+        required: false
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/feature_request.yml]: All the files are supposed to have a line at the end 😅

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -13,3 +13,19 @@ fixes # <!-- Creating an issue before an Pull Request never hurts -->
...
+## Description
+
+- [ ] I have followed the [code of conduct](https://github.com/krishdevdb/discord-plutus/blob/main/contributing.md)
```
Comment (ghost) [inline .github/pull_request_template.md]: This link needs to be updated

File: .github/pull_request_template.md (hunk 1 above)
Comment (Sonichigo) [reply .github/pull_request_template.md]: Correct same happened twice!

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -2,6 +2,104 @@ name: Bug Report
...
+      
+          Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+    validations:
+        required: false
+=======
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["💻 aspect: code", "🛠 goal: fix", "🚦 status: awaiting triage"]
 assignees:
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Is this duplicate to the one above?

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 6]
```diff
@@ -2,6 +2,60 @@ name: Documentation
...
+  - type: textarea
+    attributes:
+        label: Anything else?
+        description: |
+          Links? References? Anything that will give us more context about the issue you are encountering!
+      
+          Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+    validations:
+        required: false
+=======
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/documentation.yml]: Here's this too

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 7]
```diff
@@ -2,6 +2,60 @@ name: Feature Request
...
+  - type: textarea
+    attributes:
+        label: Anything else?
+        description: |
+          Links? References? Anything that will give us more context about the issue you are encountering!
+      
+          Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+    validations:
+        required: false
+=======
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/feature_request.yml]: Once again 😅

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -13,3 +13,5 @@ fixes # <!-- Creating an issue before an Pull Request never hurts -->
 ## How Has This Been Tested?
 
 ## Screenshots (if appropriate):
+
+=======
```
Comment (ghost) [inline .github/pull_request_template.md]: 😅

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Can you mark resolved to the reviews you have fixed?

**Comment by Sonichigo** [comment]
Done!! Anymore changes do just and a request that if all is good then merge it tomorrow😅

**Review by ghost — APPROVED** [review]

**Pull request merged**